rpc/trace_replay_tx/trace_replay.py

```diff
@@ -0,0 +1,154 @@
+
```

Comment:
I suggest to split this file into multiple pieces:
- one main module under project root
- some modules e.g. named `block_scan` (containing the logic to scan a range of blocks and apply a generic function) and `api.py` (containing JSON content for `eth_getBlockByNumber` and `trace_replayTransaction`) in `replay` folder
- one common module `jsonrpc.py` containing `make_request` function (this must be in common with existing `player` module, which does exactly the same thing